packages/create-plugin/templates/datasource/provisioning/datasources/datasources.yml

```
@@ -0,0 +1,11 @@
+apiVersion: 1
+
+datasources:
+  - name: Basic Example
```

Let's use template variables here as well.
```suggestion
  - name: '{{ pluginName }}'
```